workspace/automate_scripts/server_setup/install_containers.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -eux
+
+echo "Login to hub.myenterpriselicense.hpe.com"
+sudo docker login -u jiefu.zhu@tu-dresden.de -p hpe_eval hub.myenterpriselicense.hpe.com
```

Comment:
Do we want to maybe create an extra gmail address for swarm learning and register it with hpe so we can leave personal email out here?

Reply:
It's ok to leave mine here since we will be using this email address also as IT support for our whole kathlab repo

Reply:
Yes I agree we should do this. @kevinxpfeiffer Could you create one Gmail with katherlab_swarm as the name?

Reply:
Yes I do it immediately